integreat_cms/cms/views/pages/page_actions.py

```diff
@@ -827,3 +828,45 @@ def render_mirrored_page_field(request, region_slug, language_slug):
             "page_form": page_form,
         },
     )
+
+
```

Comment:
```suggestion

@require_POST
```
Since this view changes database contents, we need to protect it against CSRF-attacks, otherwise attackers could send refresh-links to users (or redirect to them from their site) and mark pages as up to date which are really outdated (which would be catastrophic). The simple way to prevent this is just to convert it to a post-only view, because then Django automatically requires the CSRF-token.